core/App/localization/fr/index.ts

```diff
@@ -1,98 +1,197 @@
-    "IncorrectPin": "Broche incorrecte",    
-  }, 
-  "ContactDetails": {
-    "Created": "Créé",
-    "ConnectionState": "État de la connexion"
-  },
-  "CredentialDetails": {
-    "Id": "Identifiant:",
-    "CreatedAt": "Créé à:"
-  },
-  "Home": {
-    "Notifications": "Notifications",
-    "NoNewUpdates": "Pas de nouvelles mises à jour"
-  },
-  "Scan":{
-    "SuccessfullyAcceptedConnection":"Connexion acceptée avec succès",
-    "AcceptingConnection":"Accepter la connexion",
-    "ConnectionRecordIdNotFound":"ID d'enregistrement de connexion introuvable"
-  },
-  "CredentialOffer": {
-    "ThisIsTakingLongerThanExpected": "Cela prend plus de temps que prévu. Revenez plus tard pour vos nouvelles informations d'identification.",
-    "RejectThisCredential?": "Refuser cet identifiant ?",
-    "AcceptingCredential": "Acceptation des informations d'identification",
-    "SuccessfullyAcceptedCredential": "Informations d'identification acceptées avec succès",
-    "RejectingCredential":"Rejeter les informations d'identification",
-    "SuccessfullyRejectedCredential":"Identifiant rejeté avec succès"
-  },
-  "ProofRequest": {
-    "OfferDelay": "Retard de l'offre",
-    "RejectThisProof?": "Rejeter cette preuve ?",
-    "AcceptingProof": "Accepter la preuve",
-    "SuccessfullyAcceptedProof": "Preuve acceptée avec succès",
-    "ProofNotFound":"Preuve introuvable",
-    "RequestedCredsNotFound":"Crédits demandés non trouvés",
-    "RejectingProof":"Rejeter la preuve"
-  },
-  "Settings": {
-    "Version": "Version",
-    "AMA-RNVersion": "AMA-RN Version",
-    "VersionString": "0.0.1",
-    "AMA-RNVersionString": "0.1.0",
-    "AppPreferences": "Préférences de l'application",
-    "AboutApp": "À propos de l'application",
-    "Language": "Langue",
-  },
-  "TabStack": {
-    "Home": "Accueil",
-    "Contacts": "Contacts",
-    "Scan": "Analyse",
-    "Credentials": "Crédits",
-    "Settings": "Paramètres"
-  },
-  "QRScanner": {
-    "PermissionToUseCamera": "Autorisation d'utiliser la caméra",
-    "WeNeedYourPermissionToUseYourCamera": "Nous avons besoin de votre permission pour utiliser votre appareil photo",
-    "Ok": "D'accord",
-  }
+    "Global": {
+        "EnterPin": "Entrer le NIP",
+        "6DigitPin": "NIP à 6 chiffres",
+        "Submit": "Soumettre",
+        "NoneYet!": "Votre portefeuille est vide. Vos justificatifs acceptés seront ajoutés ici.",
+        "Cancel": "Annuler",
+        "Confirm": "Confirmer",
+        "Accept": "Accepter",
+        "Reject": "Rejeter",
+        "Share": "Partager",
+        "Decline": "Refuser",
+        "Back": "Retour",
+        "Next": "Suivant",
+        "Continue": "Continuer",
+        "Info": "Information",
+        "ThisDecisionCannotBeChanged": "Cette décision ne peut être modifiée.",
+        "Failure": "Échec",
+        "Success": "Succès",
+        "SomethingWentWrong": "Un problème est survenu",
+        "Done": "Terminé",
+        "Skip": "Ignorer",
+        "View": "Voir",
```

Comment:
Consulter or Visualiser or Afficher would be more accurate, but knowing when this message will be displayed would help me to choose the right term.